src/list.rs

```diff
@@ -66,6 +106,10 @@ fn handle_list(app_state: &mut AppState, stdout: &mut StdoutLock) -> Result<()>
                             return Ok(());
                         }
                     }
+                    KeyCode::Char('s') | KeyCode::Char('/') => {
+                        list_state.message.push_str("search:|");
```

Comment:
This can be replaced with `apply_search_query`. It should return early when `search_query` is empty.

Reply:
done